CODEOWNERS

```diff
@@ -32,15 +32,18 @@
 /rising/transforms/compose.py @mibaumgartner
 /rising/transforms/crop.py @mibaumgartner
 /rising/transforms/format.py @mibaumgartner
-/rising/transforms/intensity.py @justusschock @mibaumgartner
+/rising/transforms/intensity.py @justusschock @mibaumgartner @weningerleon
 /rising/transforms/kernel.py @mibaumgartner
 /rising/transforms/spatial.py @justusschock @mibaumgartner
 /rising/transforms/utility.py @justusschock @mibaumgartner
+/rising/transforms/painting.py @weningerleon
```

Comment:
The CODEOWNERS file is basically not for code ownership (name is a bit misleading), but for automated PR review requests. This means if you register yourself here as an owner, you probably will get automated requests to review PRs for these files.

The actual code ownership is handled based on commits and the associated accounts.